keyboards/handwired/wabi/info.json

```diff
@@ -0,0 +1,12 @@
+{
+    "keyboard_name": "Wabi", 
+    "url": "www.montsinger.net", 
+    "maintainer": "Montsinger", 
+    "width": 7.25, 
+    "height": 6, 
+    "layouts": {
+        "LAYOUT": {
+            "layout": [{"label":"Esc", "x":1.25, "y":-4}, {"x":2.25, "y":-4}, {"label":"1", "x":2.25, "y":-4}, {"label":"2", "x":3.25, "y":-4}, {"label":"3", "x":4.25, "y":-4}, {"label":"4", "x":5.25, "y":-4}, {"label":"5", "x":6.25, "y":-4}, {"label":"Tab", "x":0.75, "y":-3.0, "w":1.5}, {"label":"Q", "x":2.25, "y":-3.0}, {"label":"W", "x":3.25, "y":-3.0}, {"label":"E", "x":4.25, "y":-3.0}, {"label":"R", "x":5.25, "y":-3.0}, {"label":"T", "x":6.25, "y":-3.0}, {"label":"Caps Lock", "x":0.5, "y":-2, "w":1.75}, {"label":"A", "x":2.25, "y":-2}, {"label":"S", "x":3.25, "y":-2}, {"label":"D", "x":4.25, "y":-2}, {"label":"F", "x":5.25, "y":-2}, {"label":"G", "x":6.25, "y":-2}, {"label":"Shift", "x":0, "y":-1.0, "w":2.25}, {"label":"Z", "x":2.25, "y":-1.0}, {"label":"X", "x":3.25, "y":-1.0}, {"label":"C", "x":4.25, "y":-1.0}, {"label":"V", "x":5.25, "y":-1.0}, {"label":"B", "x":6.25, "y":-1.0}, {"x":0, "y":0, "w":1.25}, {"x":1.25, "y":0, "w":1.25}, {"x":4, "y":0, "w":1.25}, {"x":5.25, "y":0}, {"x":6.25, "y":0}, {"label":"6", "x":-8, "y":1}, {"label":"7", "x":-7.0, "y":1}, {"label":"8", "x":-6, "y":1}, {"label":"9", "x":-5.0, "y":1}, {"label":"0", "x":-4, "y":1}, {"label":"-", "x":-3.0, "y":1}, {"label":"=", "x":-2, "y":1}, {"label":"Backspace", "x":-1.0, "y":1, "w":2}, {"label":"Y", "x":-8, "y":2}, {"label":"U", "x":-7.0, "y":2}, {"label":"I", "x":-6, "y":2}, {"label":"O", "x":-5.0, "y":2}, {"label":"P", "x":-4, "y":2}, {"label":"[", "x":-3.0, "y":2}, {"label":"]", "x":-2, "y":2}, {"label":"|", "x":-1.0, "y":2, "w":1.5}, {"label":"H", "x":-8, "y":3}, {"label":"J", "x":-7.0, "y":3}, {"label":"K", "x":-6, "y":3}, {"label":"L", "x":-5.0, "y":3}, {"label":";", "x":-4, "y":3}, {"label":"'", "x":-3.0, "y":3}, {"label":"Enter", "x":-2, "y":3, "w":2.25}, {"label":"N", "x":-8, "y":4}, {"label":"M", "x":-7.0, "y":4}, {"label":",", "x":-6, "y":4}, {"label":".", "x":-5.0, "y":4}, {"label":"?", "x":-4, "y":4}, {"label":"Shift", "x":-3.0, "y":4, "w":2}, {"x":-1.0, "y":4}, {"x":-8, "y":5}, {"x":-7.0, "y":5}, {"x":-6, "y":5}, {"x":-2, "y":5}, {"x":-1.0, "y":5}, {"x":0, "y":5}]
```

Comment:
This is completely messed up:

![image](https://user-images.githubusercontent.com/18669334/89569268-4f81c080-d7d9-11ea-8e41-2ef1fa7bd067.png)

```suggestion
            "layout": [{"label":"Esc", "x":1.25, "y":0}, {"label":"1", "x":2.25, "y":0}, {"label":"2", "x":3.25, "y":0}, {"label":"3", "x":4.25, "y":0}, {"label":"4", "x":5.25, "y":0}, {"label":"5", "x":6.25, "y":0}, {"label":"6", "x":8.75, "y":0}, {"label":"7", "x":9.75, "y":0}, {"label":"8", "x":10.75, "y":0}, {"label":"9", "x":11.75, "y":0}, {"label":"0", "x":12.75, "y":0}, {"label":"-", "x":13.75, "y":0}, {"label":"=", "x":14.75, "y":0}, {"label":"Backspace", "x":15.75, "y":0, "w":2}, {"label":"Tab", "x":0.75, "y":1, "w":1.5}, {"label":"Q", "x":2.25, "y":1}, {"label":"W", "x":3.25, "y":1}, {"label":"E", "x":4.25, "y":1}, {"label":"R", "x":5.25, "y":1}, {"label":"T", "x":6.25, "y":1}, {"label":"Y", "x":8.75, "y":1}, {"label":"U", "x":9.75, "y":1}, {"label":"I", "x":10.75, "y":1}, {"label":"O", "x":11.75, "y":1}, {"label":"P", "x":12.75, "y":1}, {"label":"[", "x":13.75, "y":1}, {"label":"]", "x":14.75, "y":1}, {"label":"\\", "x":15.75, "y":1, "w":1.5}, {"label":"Caps Lock", "x":0.5, "y":2, "w":1.75}, {"label":"A", "x":2.25, "y":2}, {"label":"S", "x":3.25, "y":2}, {"label":"D", "x":4.25, "y":2}, {"label":"F", "x":5.25, "y":2}, {"label":"G", "x":6.25, "y":2}, {"label":"H", "x":8.75, "y":2}, {"label":"J", "x":9.75, "y":2}, {"label":"K", "x":10.75, "y":2}, {"label":"L", "x":11.75, "y":2}, {"label":";", "x":12.75, "y":2}, {"label":"'", "x":13.75, "y":2}, {"label":"Enter", "x":14.75, "y":2, "w":2.25}, {"label":"Shift", "x":0, "y":3, "w":2.25}, {"label":"Z", "x":2.25, "y":3}, {"label":"X", "x":3.25, "y":3}, {"label":"C", "x":4.25, "y":3}, {"label":"V", "x":5.25, "y":3}, {"label":"B", "x":6.25, "y":3}, {"label":"N", "x":8.75, "y":3}, {"label":"M", "x":9.75, "y":3}, {"label":",", "x":10.75, "y":3}, {"label":".", "x":11.75, "y":3}, {"label":"/", "x":12.75, "y":3}, {"label":"Shift", "x":13.75, "y":3, "w":2.75}, {"label":"Up", "x":16.5, "y":3}, {"label":"Ctrl", "x":0, "y":4, "w":1.25}, {"label":"Win", "x":1.25, "y":4, "w":1.25}, {"label":"Alt", "x":4, "y":4, "w":1.25}, {"label":"Space", "x":5.25, "y":4}, {"label":"Page Down", "x":6.25, "y":4, "w":1.25}, {"label":"Enter", "x":8.5, "y":4, "w":1.25}, {"label":"Space", "x":9.75, "y":4}, {"label":"Backspace", "x":10.75, "y":4, "w":1.25}, {"label":"Left", "x":15.5, "y":4}, {"label":"Down", "x":16.5, "y":4}, {"label":"Right", "x":17.5, "y":4}]
```